EXERCISE-3.md

```diff
@@ -472,10 +472,30 @@ On the same page where the ONOS topology view is shown:
 * Press `A` multiple times until you see link stats, in either 
   packets/seconds (pps) or bits/seconds.
 
+> Only the hosts that have been pinged successfully at this point in mininet will
+>be visible.
+
 Link stats are derived by ONOS by periodically obtaining the port counters for
 each device. ONOS internally uses gNMI to read port information, including
 counters.
 
+> All of the devices, ports, portstats, links, hosts, flows and groups can be
+>displayed through the various views in the web UI.
+
+#### 7a. View Pipeconf data in the web UI
+
+In the ONOS topology view click on one of the Stratum switches (e.g `device:leaf1`)
+and the Device Details panel appears. In that panel click on the Pipeconf icon
+(the last one), to open the Pipeconf view for that device.
```

Comment:
It would be  good to attach a screenshot of the panel and pipeconf button to click